### PR TITLE
Make redirects of the logos in headers consistent

### PR DESCRIPTION
**Pull request by PanLiszka**
- Added redirectToBoard patch which changes redirects of the "Dzienniczek Vulcan" and "eduVULCAN" logos to be more consistant

### CODE REVIEW

File: patches/redirectToBoard.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    if (getLogoElement()) {
+        observer.disconnect();
+        redirectToBoard();
+    }
+});
+
+function redirectToBoard() {
+    const logoElement = getLogoElement();
+    if (!!window.location.hostname.match(/^(dziennik-)?wiadomosci.*/)) {
+        const url = !!window.location.hostname.match(
```
Comment (banocean) [inline patches/redirectToBoard.js:23]: two times checked for the same thing and still adding 2nd option that isn't even possible

File: patches/redirectToBoard.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        logoElement.onclick = () =>
+            document.querySelector(".tablica a").click();
+    }
+}
+
+if (getLogoElement()) {
+    redirectToBoard();
+} else
+    logoObserver.observe(document.body, {
+        characterData: true,
```
Comment (banocean) [inline patches/redirectToBoard.js:58]: why?

File: patches/redirectToBoard.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        childList: true,
+        attributes: true,
```
Comment (banocean) [inline patches/redirectToBoard.js:60]: also why?

**Review by banocean — CHANGES_REQUESTED** [review]

**Review by banocean — CHANGES_REQUESTED** [review]

**Review by PanLiszka — COMMENTED** [review]
you haven't seen it, ok?

File: patches/redirectToBoard/script.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ const isEduVulcan = !window.location.hostname.startsWith("dziennik");
 const isMobile = window.innerWidth < 1024;
 
 const getLogoElement = () =>
-    document.querySelector(".header__logo-product a").firstChild;
+    document.querySelector(".header__logo-product").firstChild;
```
Comment (PanLiszka) [inline patches/redirectToBoard/script.js:5]: you haven't seen it, ok?

**Review by banocean — APPROVED** [review]
LGTM!

**Pull request merged**